src/components/WalletSelector/WalletSelector.tsx

```diff
@@ -121,8 +121,8 @@ function WalletSelector({ connectionMode = AUTH, queryRef }: Props) {
       if (detectedError.code === 'GALLERY_SERVER_ERROR') {
         if (isNotEarlyAccessError(detectedError.message)) {
           return {
-            heading: 'Authorization Error',
-            body: "We weren't able to locate a Membership Card in your wallet. Gallery is in closed beta and requires one for access. You can buy one on the [secondary market](https://opensea.io/collection/gallerygeneralmembershipcards) or visit our [FAQ](https://gallery-so.notion.site/Gallery-FAQ-b5ee57c1d7f74c6695e42c84cb6964ba) for more info.",
+            heading: account?.toLowerCase(),
+            body: 'Your wallet address is not on the **Early Access Allowlist**. To get onto the allowlist, visit our [FAQ](https://gallery-so.notion.site/Gallery-FAQ-b5ee57c1d7f74c6695e42c84cb6964ba#6fa1bc2983614500a206fc14fcfd61bf) or reach out to us on [Discord](discord.gg/vBqBEH8GaM).',
```

Comment:
puts the wallet address in the header, as discussed with jess 

<img width="313" alt="image" src="https://user-images.githubusercontent.com/12162433/172962150-362a52db-0918-4180-8f91-58060272b717.png">